### ISSUE TITLE
Meaning of colours of the dots on the maps

### ISSUE THREAD

**Issue by gbif-portal** [gbif/portal-feedback#563]
**Meaning of colours of the dots on the maps**

In the  dataset EBCC Atlas of European Breeding Birds there many dots in various colours. This is also the case in many other datasets.  I cannot  find a legend or explanation, which would be helpful to interpret the map.
Is it possible to add a legenda or do I overlook something?

The dataset can be found here:
https://www.gbif.org/dataset/c779b049-28f3-4daf-bbf4-0a40830819b6

-----
fbitem-datasetc779b049-28f3-4daf-bbf4-0a40830819b6
User provided contact info: jeroen.creuwels@naturalis.nl
System: Chrome 61.0.3163 / Windows 7 0.0.0
Referer: https://www.gbif.org/dataset/c779b049-28f3-4daf-bbf4-0a40830819b6
Window size: width 1891 - height 862
[API log](http://elk.gbif.org:5601/app/kibana#/discover/UAT-Varnish-403s?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2017-10-18T12:02:12.534Z',mode:absolute,to:'2017-10-18T12:08:12.534Z'))&_a=(columns:!(request,response,clientip),filters:!(),index:'prod-varnish-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E499%20AND%20(request:%22%2F%2Fapi.gbif.org%22)')),sort:!('@timestamp',desc))&indexPattern=uat-varnish-*&type=histogram)
[Site log](http://elk.gbif.org:5601/app/kibana#/discover/UAT-Varnish-403s?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2017-10-18T12:02:12.534Z',mode:absolute,to:'2017-10-18T12:08:12.534Z'))&_a=(columns:!(request,response,clientip),filters:!(),index:'prod-varnish-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E399%20AND%20(request:%22%2F%2Fdemo.gbif.org%22)')),sort:!('@timestamp',desc))&indexPattern=uat-varnish-*&type=histogram)

**Comment by MattBlissett** [comment]
Yes, this is missing. The colours for the classic view are specified here: https://github.com/gbif/maps/blob/master/mapnik-server/src/main/node/cartocss/classic-dot.mss

**Comment by MortenHofft** [comment]
It isn't there and it is a very reasonable request. Might be something to add as a capabilities API of a sort? What intervals are color coded how? Basically a `/legend?style=classic-dot`
